CLI/PkModeling.xml

```diff
@@ -126,7 +126,7 @@
       <description><![CDATA[Input 4D Image (txyz)]]></description>
     </image>
 
-    <image type="label">
```

Comment:
I think this one and the following were removed by mistake - these inputs are labels, and it is important to have these attributes to constrain the selection of the input nodes in the GUI.